### PR TITLE
[1216][ADD] sale_order_value_proposal_by_partner_shipping

### PR DESCRIPTION
**Pull request by TimLai125**
#[1216](https://www.quartile.co/web#id=1216&model=project.task&view_type=form&menu_id=)

### CODE REVIEW

File: sale_order_partner_shipping_propose/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Sale Order Delivery Address Propose",
+    "version": "12.0.1.0.0",
+    "author": "Quartile Limited",
+    "website": "https://www.quartile.co",
+    "category": "Sales",
+    "license": "AGPL-3",
```
Comment (yostashiro) [inline sale_order_partner_shipping_propose/__manifest__.py]:
```suggestion
    "license": "LGPL-3",
```

**Review by yostashiro — CHANGES_REQUESTED** [review]
@TimLai125 Could we change the name to "sale_order_value_proposal_by_partner_shipping", or is this too long?

**Comment by TimLai125** [comment]
> @TimLai125 Could we change the name to "sale_order_value_proposal_by_partner_shipping", or is this too long?

I think it is still acceptable. Let me update the name of the module

File: sale_order_value_proposal_by_partner_shipping/models/res_partner.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Copyright 2020 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    warehouse_id = fields.Many2one("stock.warehouse", string="Warehouse")
```
Comment (yostashiro) [inline sale_order_value_proposal_by_partner_shipping/models/res_partner.py]: Just came to think that this field should actually be a property field to make it more generic.  What do you think?

**Review by yostashiro — APPROVED** [review]
LGTM!

**Pull request merged**